### DISCUSSION TITLE
Equivalent approach for drawable and color xml?

### DISCUSSION THREAD

**Discussion by mochadwi** [B3nedikt/restring#108] [Q&A] [answered]
Is it possible to do this for other than `getString` as well?

**Comment by B3nedikt** [comment, marked as answer]
Yes, you would
- need to overwrite the correct methods in [RestringResources](https://github.com/B3nedikt/restring/blob/3acceded6fb40d4cb5634b27d4b01415276b9c67/restring/src/main/java/dev/b3nedikt/restring/internal/RestringResources.kt).
- then define new repositories for it e.g. ColorsRepository, DrawablesRepository.
- Extend the Restring facade object with new methods for changing colors & drawables
- And maybe extend the reword library, if you need to change these resources without restarting the activity

Quite a bit of work, which I don't have time for now ;)
But feel free to implement it, I take PRs :)
